config/services.yml

```diff
@@ -143,3 +143,8 @@
   csproj_path: 'GetIntoTeachingApi/GetIntoTeachingApi.csproj'
   docsets:
     - path: 'docs'
+- name: "ITT Mentor Services"
+  repo_name: 'DFE-Digital/itt-mentor-services'
+  language: 'ruby'
+  docsets:
+    - path: 'docs'
```

Comment:
Our ADRs are in a separate directory, so I'm wondering if we want to include that path too?

Similar to what the 'Register trainee teachers' service does:

https://github.com/DFE-Digital/teacher-services-tech-docs/blob/9ceaa7a3bf032fedb4b8db4bad930dd330276d74/config/services.yml#L15-L20

Reply:
Resolved in [4635543](https://github.com/DFE-Digital/teacher-services-tech-docs/pull/49/commits/4635543aec47b8765eafff885a7e3763482a33a8)